build.gradle

```diff
@@ -176,20 +176,15 @@ dependencies {
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: versions.springBoot
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-actuator', version: versions.springBoot
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-aop', version: versions.springBoot
-//  compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: '2.1.1.RELEASE'
-
   compile group: 'io.rest-assured', name: 'rest-assured', version: '3.3.0'
-  compile group: 'io.github.openfeign', name: 'feign-jackson', version: '10.1.0'
 
   compile group: 'io.springfox', name: 'springfox-swagger2', version: versions.springfoxSwagger
   compile group: 'io.springfox', name: 'springfox-swagger-ui', version: versions.springfoxSwagger
 
-  compile group: 'uk.gov.hmcts.reform', name: 'health-spring-boot-starter', version:'0.0.4'
```

Comment:
needed for helm charts liveness indicator. i.e. adds a [management]/health/liveness.
 
See Readme in https://github.com/hmcts/health-spring-boot-starter